scripts/ci/tags.yaml

```diff
@@ -107,261 +105,288 @@ ci_samples_net: &ci_net
+    - nrf/cmake/
     - nrf/include/bl*
-    - nrf/include/fprotect.h
     - nrf/include/dfu/*suit*
+    - nrf/include/fprotect.h
     - nrf/include/sdfw/sdfw_services/
-    - nrf/cmake/
-    - nrf/sysbuild/
     - nrf/lib/dk_buttons_and_leds/
-    - nrf/subsys/suit/
+    - nrf/samples/suit/
     - nrf/subsys/dfu/
+    - nrf/subsys/mgmt/suitfu/
     - nrf/subsys/nrf_rpc/
     - nrf/subsys/nrf_security/
     - nrf/subsys/sdfw_services/
-    - nrf/subsys/mgmt/suitfu/
+    - nrf/subsys/suit/
+    - nrf/sysbuild/
     - nrf/tests/subsys/suit/
-    - nrf/samples/suit/
-    - modules/lib/suit-processor/
-    - modules/lib/suit-generator/
+    - zephyr/cmake/
+    - zephyr/share/
 
 ci_samples_nfc:
   files:
-    - nrf/subsys/nfc/
     - nrf/include/nfc/
     - nrf/lib/st25r3911b/
-    - nrf/samples/nfc/
-    - nrf/samples/bluetooth/peripheral_nfc_pairing/
     - nrf/samples/bluetooth/central_nfc_pairing/
     - nrf/samples/bluetooth/peripheral_hids_keyboard/
+    - nrf/samples/bluetooth/peripheral_nfc_pairing/
+    - nrf/samples/nfc/
+    - nrf/subsys/nfc/
     - nrfxlib/nfc/
 
 Sidewalk:
   files:
-    - nrf/subsys/bootloader/
-    - nrf/subsys/partition_manager/
-    - nrf/subsys/dfu/
-    - nrf/modules/mcuboot/
-    - nrf/subsys/fw_info/
-    - nrf/subsys/pcd/
-    - nrf/subsys/bluetooth/controller/
-    - nrf/subsys/nrf_security/
-    - nrf/subsys/mpsl/
+    - bootloader/mcuboot/
+    - modules/crypto/mbedtls/
+    - modules/crypto/oberon-psa-crypto/
+    - modules/crypto/tinycrypt/
+    - modules/lib/open-amp/open-amp/
+    - modules/lib/zcbor/
     - nrf/applications/ipc_radio/
     - nrf/drivers/entropy/
     - nrf/drivers/hw_cc3xx/
     - nrf/drivers/mpsl/
     - nrf/lib/dk_buttons_and_leds/
     - nrf/lib/fatal_error/
     - nrf/lib/fprotect/
+    - nrf/modules/mcuboot/
+    - nrf/subsys/bluetooth/controller/
+    - nrf/subsys/bootloader/
+    - nrf/subsys/dfu/
+    - nrf/subsys/fw_info/
+    - nrf/subsys/mpsl/
+    - nrf/subsys/nrf_security/
+    - nrf/subsys/partition_manager/
+    - nrf/subsys/pcd/
+    - nrfxlib/crypto/
+    - sidewalk/
     - zephyr/include/dfu/
     - zephyr/include/mgmt/mcumgr/
+    - zephyr/samples/bluetooth/hci_ipc/
     - zephyr/soc/nordic/
     - zephyr/subsys/dfu/
-    - zephyr/subsys/settings/
-    - zephyr/subsys/mgmt/mcumgr/
-    - zephyr/samples/bluetooth/hci_ipc/
     - zephyr/subsys/fs/
     - zephyr/subsys/ipc/
     - zephyr/subsys/mem_mgmt/
     - zephyr/subsys/mgmt/
+    - zephyr/subsys/mgmt/mcumgr/
     - zephyr/subsys/net/
     - zephyr/subsys/pm/
     - zephyr/subsys/random/
     - zephyr/subsys/settings/
+    - zephyr/subsys/settings/
     - zephyr/subsys/shell/
     - zephyr/subsys/storage/
     - zephyr/subsys/usb/
-    - nrfxlib/crypto/
-    - modules/crypto/mbedtls/
-    - modules/crypto/oberon-psa-crypto/
-    - modules/crypto/tinycrypt/
-    - modules/lib/open-amp/open-amp/
-    - modules/lib/zcbor/
-    - bootloader/mcuboot/
-    - sidewalk/
 
 find_my:
   files:
     - bootloader/mcuboot/
```

Comment:
Can we also add the build system dependencies to `find-my` please?

Reply:
Will add.